source/Userdocs/Software/NeuroMLlite.md

```diff
@@ -1 +1,37 @@
 # NeuroMLlite
+
+NeuroMLlite is a common framework for reading/writing/generating network specifications which builds on NeuroML 2.
```

Comment:
This is confusing: it speaks about a framework and specification and then describes a tool? Like with NeuroML where we need to distinguish between the standard/spec and the tools, we need to do this here too.